### DISCUSSION TITLE
Error When Installing

### DISCUSSION THREAD

**Discussion by ibnbob** [antlr/antlr4-tools#4] [Q&A]
I get the following error when trying to to install ANTLR4. I did make sure that I updated pip. What else am I missing?

[08:22:02] ~/tools >> pip install atlr4-tools
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement atlr4-tools (from versions: none)
ERROR: No matching distribution found for atlr4-tools

**Comment by ericvergnaud** [comment]
Please check typo...

**Comment by parrt** [comment]
Hi @ibnbob I think you needed "antlr4-tools" :)
